src/styles/container.scss

```diff
@@ -0,0 +1,8 @@
+.container {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  width: 1024px;
```

Comment:
don't hardcode your container width, use max-width with calculation based on card-width, gap between cards and number of cards